### PR TITLE
Modal profile body

### PR DESCRIPTION
**Pull request by arse0039**
I created the modal profile body with a fading animation. 
We will still need to structure it as we see fit, visually.

### CODE REVIEW

File: frontend/src/pages/aboutPage.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { useState } from 'react';
 import '../styles/aboutPage.css';
 import brandonImage from '../assets/BArsenault.jpg';
 import davidImage from '../assets/DClaphan.jpg';
-import AboutModal from '../components/aboutModal';
+import AboutModal from '../components/AboutModal';
```
Comment (davidclaphan) [inline frontend/src/pages/aboutPage.tsx]: I am getting an error from this I think it needs to be lowercase "a" in '/aboutModal'

**Review by davidclaphan — CHANGES_REQUESTED** [review]
1 question - is the index.html page just for importing fonts? 

1 change request - commented below about lowercase "a" in import statement

File: frontend/src/pages/aboutPage.tsx (hunk 1 above)
Comment (arse0039) [reply frontend/src/pages/aboutPage.tsx]: It does. I tried to change the filename to be inline with our naming conventions but for some reason that change wouldn't push.

Comment (arse0039) [reply frontend/src/pages/aboutPage.tsx]: fixed!

**Review by davidclaphan — APPROVED** [review]

**Pull request merged**